package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hipo/react-ui-toolkit",
-  "version": "1.0.0-alpha.5.1.0",
+  "version": "1.0.0-alpha.5.1.1",
```

Comment:
Version change shouldn't have been done here. It is generally better if the person who is preparing the new published version to update the version of the package after all the changes are merged. Also version changes should be reflected on the package-lock file as well.